apps/frontend/src/pages/projects/libs/components/project-menu/project-menu.tsx

```diff
@@ -8,6 +11,7 @@ type Properties = {
 
 const ProjectMenu = ({ onDelete, onEdit }: Properties): JSX.Element => {
 	const { isOpened, onClose, onOpen } = usePopover();
+	const { userPermissions } = useAppSelector(({ auth }) => auth);
```

Comment:
We need to pass permissions through props here, this should be a simple component